frontend/src/app/commonComponents/TestResultsList.tsx

```diff
@@ -78,7 +78,9 @@ const pxpAppResultList = (
 ) => {
   return (
     <>
-      <h2 className="font-heading-sm">{setDiseaseName(diseaseName, t)}</h2>
+      <h2 className="font-heading-sm">
+        {setDiseaseName(diseaseName, t)} result
```

Comment:
I'm afraid we need a Spanish translation for this - we may already have one, it would be in es.ts if so.

Reply:
I do not see one for "Disease Result" in either. Is there a place I need to request that translation?

Reply:
I can Slack you what I had to do for print modal translations when I worked on that last and need translations!

Reply:
We do have the translations for the diseases here: https://github.com/CDCgov/prime-simplereport/blob/14862ff3925a413c732c6bc9bff8e5cc8f297591/frontend/src/lang/es.ts#L39-L39 and they're already being used in `setDiseaseName`. It's just the word "result" that we need to make sure to translate. We can definitely put a translation request in, but in the meantime to unblock this PR I think it's fine to add a new entry in es.ts and en.ts (maybe resultLiteral? Somewhere inside of the [testResult](https://github.com/CDCgov/prime-simplereport/blob/14862ff3925a413c732c6bc9bff8e5cc8f297591/frontend/src/lang/es.ts#L260-L260) object) and add "resultado" for the Spanish translation. That's what we're using elsewhere already.

Reply:
Sent in the request for the translations. Will create a new PR once those come in!